test/performance/range/views/view_translate_2D_benchmark.cpp

```diff
@@ -11,22 +11,22 @@
 #include <benchmark/benchmark.h>
 
 #include <seqan3/alphabet/nucleotide/dna4.hpp>
-#include <seqan3/range/views/join.hpp>
 #include <seqan3/range/views/to.hpp>
 #include <seqan3/range/views/translate.hpp>
 #include <seqan3/range/views/translate_join.hpp>
 #include <seqan3/test/performance/sequence_generator.hpp>
 #include <seqan3/test/seqan2.hpp>
+#include <seqan3/utility/views/join_with.hpp>
 
 #ifdef SEQAN3_HAS_SEQAN2
-#include <seqan/sequence.h>
 #include <seqan/seq_io.h>
+#include <seqan/sequence.h>
 #include <seqan/translation.h>
 #endif
 
 // Tags used to define the benchmark type
 struct baseline_tag{}; // Baseline where view is applied and only iterating the output range is benchmarked
-struct translate_tag{}; // Benchmark view_translate followed by seqan3::views::join
+struct translate_tag{}; // Benchmark view_translate followed by std::views::join
```

Comment:
Question, we don't want to benchmark with join_with?

Reply:
I think we are comparing the pipe `seqan3::views::translate | std::views::join` against `seqan3::views::translate_join`.
`std::views::join_with` should only be used if you need a delimiter, otherwise `std::views::join` does the job.